### DISCUSSION TITLE
Resume Review Amrutha

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/resume-pitch#105] [Q&A]
[amrt.pdf](https://github.com/user-attachments/files/17109475/amrt.pdf)
[ATS Version of CV (30).docx](https://github.com/user-attachments/files/17109476/ATS.Version.of.CV.30.docx)

**Comment by PranjalTrivedi** [comment]
As per my point of view, Simple resume with no style is always ATS friendly. I tried and tested it.

**Comment by PranjalTrivedi** [comment]
@sanjay-kv could you please check my git profile.
